Source/WebKit/WebProcess/WebPage/Cocoa/WebPageCocoa.mm

```diff
@@ -366,7 +369,7 @@
 
 void WebPage::getContentsAsAttributedString(CompletionHandler<void(const WebCore::AttributedString&)>&& completionHandler)
 {
-    completionHandler(attributedString(makeRangeSelectingNodeContents(*m_page->mainFrame().document())));
+    completionHandler(attributedString(makeRangeSelectingNodeContents(*dynamicDowncast<LocalFrame>(m_page->mainFrame())->document())));
```

Comment:
null check

Reply:
since we're passing a reference, that would mean we skip calling completion, which seems bad